scripts/HPC_chatbot_preprocessor/parsed_mds/generic/connecting/connecting_paragraph_2.txt

```diff
@@ -0,0 +1,18 @@
+Connection restrictions
+Since March 20th 2020, restrictions are in place that limit from where
```

Comment:
this can be temoved i guess? not sure you make footnotes, but imho this is not relevant anymore